internal/cli/atlas_clusters_list.go

```diff
@@ -27,22 +27,25 @@ type atlasClustersListOpts struct {
 	*globalOpts
 	pageNum      int
 	itemsPerPage int
-	store        store.ClusterLister
+	storeAT      store.ClusterLister
+	storeOM      store.ListAllClusters
 }
 
 func (opts *atlasClustersListOpts) init() error {
+	var err error
+
 	if opts.ProjectID() == "" {
-		return errMissingProjectID
+		opts.storeOM, err = store.New()
```

Comment:
store.New() implements all the methods of available this if is big code smell that you probably just need a new contract `atlasClustersListOpts` and not multiple fields, see my previous comment